fetcher/extras/races.py

```diff
@@ -5,3 +5,9 @@ def handle_ak(res, mapping):
     res = res[0].get('features', [])
     collected = {x['attributes']['Race']: x['attributes']['CountCases'] for x in res}
     return map_attributes(collected, mapping, 'AK')
+
+def handle_mo(res, mapping):
+    # Do a quick pass to collect everything then send it to `map_attributes`
+    res = res[0].get('features', [])
+    collected = {x['attributes']['RACE']: x['attributes']['Frequency'] for x in res}
+    return map_attributes(collected, mapping, 'MO')
```

Comment:
```suggestion
    return map_attributes(collected, mapping, 'MO')

```

Reply:
I'm going to enable `Flake8`, and it'll start complaining about it

Reply:
I can amend that commit but I believe I'll have to force push. Is that okay with you?

Reply:
GitHub should let you just accept the suggestion. It'll write another commit and I can "Squash and Merge" when I merge it